src/crypto/crypto_x509.cc

```diff
@@ -57,13 +59,43 @@ void ManagedX509::MemoryInfo(MemoryTracker* tracker) const {
 }
 
 namespace {
+void AddFingerprintDigest(const unsigned char* md,
+                          unsigned int md_size,
+                          char fingerprint[3 * EVP_MAX_MD_SIZE]) {
+  unsigned int i;
+  static constexpr char hex[] = "0123456789ABCDEF";
+
+  for (i = 0; i < md_size; i++) {
+    fingerprint[3 * i] = hex[(md[i] & 0xf0) >> 4];
+    fingerprint[(3 * i) + 1] = hex[(md[i] & 0x0f)];
+    fingerprint[(3 * i) + 2] = ':';
+  }
+
+  DCHECK_GT(md_size, 0);
+  fingerprint[(3 * (md_size - 1)) + 2] = '\0';
+}
+
+MaybeLocal<Value> GetFingerprintDigest(Environment* env,
```

Comment:
this function always returns a value (undefined or onebytestring). I think MaybeLocal<> should be `Value`

Reply:
or undefined line should be `return {}`

Reply:
I'd like to keep this as `MaybeLocal<Value>` for consistency with the other methods. I do plan on iterating on this tho and likely cleaning up the implementation here in another follow on PR. There's a lot to update in all of this and I'm trying to do so very incrementally.